pkgs/by-name/se/sesh/package.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "sesh";
+  version = "be2badd29206f66803b76b62b23f6f5e05befe08";
```

Please use tagged version

```suggestion
  version = "0.12.0";
```